src/lib.rs

```diff
@@ -72,14 +61,31 @@ impl AsRef<[u8]> for Hash {
 #[cfg(test)]
 use rand::distributions::{Distribution, Standard};
 
+#[derive(Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct SpendKey(pub PublicKey);
+
+// Display Hash value as hex in Debug output.  consolidates 36 lines to 3 for pretty output
+// and the hex value is the same as sn_dbc_mint display of DBC IDs.
+impl fmt::Debug for SpendKey {
```

Comment:
maybe should impl fmt::Debug for Hash also?

Reply:
Yep, we have it already implemented